content/CORETutorials/JailsPluginsVMs/Plugins/MinIOPlugin.md

```diff
@@ -0,0 +1,208 @@
+
+    b. Click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i> to expand Minio details and management options.
+
+    c. Click <span class="material-icons">device_hub</span>&nbsp;**MOUNT POINTS** and ensure configured mount point matches the destination dataset you created in [First Steps](#first-steps).
+        If needed, follow the instructions in [Setting Up Jail Storage]({{< relref "SettingUpJailStorage.md" >}}) to mount the destination dataset.
+
+3. Use the **MinIO Console** to recreate the configuration settings from the S3 service MinIO deployment in the Minio plugin deployment.
+
+    a. Go to the MinIO web UI portal for the S3 service deployment and note existing configuration settings including users, groups, access keys, and all other MinIO settings.
+
+    b. Go to **Plugins** and locate the installed Minio plugin or go to **Jails** and locate the jail matching the **Jail Name** entered during [installation of the Minio plugin](#installing-the-minio-plugin).
+
+    c. Click <i class="material-icons" aria-hidden="true" title="Expand">chevron_right</i> to expand Minio details and management options.
+
+    d. Click <span class="material-icons">play_arrow</span>&nbsp;**START** to start the plugin (if needed) and then click <span class="material-icons">settings</span>&nbsp;**MANAGE** to go to the **MinIO Console** and log in.
+
+    e. Configure MinIO settings in the new deployment to match those of the existing deployment.
+
+4. Use the **MinIO Console** to manually recreate data buckets from the existing S3 service deployment in the Minio Plugin, including rules, versioning, quotas, and locks.
+    Bucket names do not need to be identical for this process, but your workflow may benefit from using identical names.
+
+### Migrating Data
+
+After installing and configuring the Minio plugin and installing the MinIO Client (`mc`), begin migrating data.
+
+1. From the Linux or WSL command line, enter the command `mc alias set NEWALIAS PATH ACCESSKEY SECRETKEY` for both the origin MinIO service deployment and the destination Minio plugin deployment.
+
+    {{< trueimage src="images/CORE/13.0/MinioClientSetAlias.png" alt="MinIO Client Set Alias" id="MinIO Client Set Alias" >}}
+
+2. Enter `mc mirror --preserve --watch SOURCE/BUCKET TARGET/BUCKET` to begin copying data to the Minio plugin deployment.
+
+    {{< trueimage src="images/CORE/13.0/MinioClientMirror.png" alt="MinIO Client Mirror" id="MinIO Client Mirror" >}}
+
+    The `mc mirror` operation with the `--watch` flag does not end when the transfer is completed, as it continually looks for new files added to the source.
+    The transfer rate also does not reach 0, as it is an average throughout the operation.
+
+3. Verify the transfer has completed successfully by comparing bucket size and items through the web UI of each deployment.
```

Comment:
past tense
```suggestion
3. Verify the transfer completed successfully by comparing bucket size and items through the web UI of each deployment.
```
I'm questioning why this is a step in this process when it is just more clarification of step 2 action. You can rewrite this and step 4 as continuation of step 2.

Reply:
Will address the numbering comment in a future commit